libs/auth/src/java_card.ts

```diff
@@ -378,10 +387,8 @@ export class JavaCard implements Card {
     // VotingWorks cert
     await this.verifyCardPrivateKey(CARD_VX_CERT.PRIVATE_KEY_ID, cardVxCert);
 
-    const user = await parseUserDataFromCert(
-      cardVxAdminCert,
-      this.jurisdiction
-    );
+    const { user, jurisdiction } = await parseUserDataFromCert(cardVxAdminCert);
+    assert(jurisdiction === vxAdminCertAuthorityCertDetails.jurisdiction);
```

Comment:
Though I'm no longer checking the jurisdiction against the machine jurisdiction in this layer, I'm still checking that the jurisdiction is consistent across certs stored on the card